frameworks/kafka/universe/config.json

```diff
@@ -40,16 +40,6 @@
             "type": "boolean",
             "default":  false
           },
-          "tls": {
```

Comment:
Oh ... wait, we haven't released this yet! I was just thinking that this was a breaking change.

On that topic ... where do we stand on the versioning of these schemas?

Reply:
That, is an excellent question.

They can't change within a major version, but they _can_ change between major versions.

Reply:
Yes, but we don't version the schemas themselves. We could consider something like: https://snowplowanalytics.com/blog/2014/05/13/introducing-schemaver-for-semantic-versioning-of-schemas/?